### ISSUE TITLE
FranceTVPluzz : not working at all

### ISSUE THREAD

**Issue by sm1090** [JUL1EN094/JUL1EN094-xbmc-addons#8]
Hi !

I noticed today that FranceTVPluzz seems completely broken although it worked perfectly recently. After it starts, I can neither play any live  stream, nor access to any replay.
Currently Freplay works though.

Would you have any hint ?

Cheers !

**Comment by JUL1EN094** [comment]
Please could u try the new version 1.1.1 and gimme a feedback ?

**Comment by sm1090** [comment]
Hi !

Thanks for this update.

I gave it a try and tested it thoroughly using a VPN to counter georestrictions. I used FranceTVPluzz on openelec (6.0.3 official version) installed on a Raspberry Pi 2.

**Live stream** worked fine for every channel except the ones associated with overseas french regions (Guyane, Martinique...). They might work from these areas however (I do not have a VPN to connect in these regions).

**Replays** were tested on two items for every channel in the list. All the tested items worked fine. I could find a minor bug : the durations appearing on the list before a video is played are always wrong. Once a video was played once, its duration is updated and the right duration is displayed.

Apart from this minor bug, there were no other issues.

Thanks again for this update !
